### DISCUSSION TITLE
Search query for individual new faces

### DISCUSSION THREAD

**Discussion by rambip** [photoprism/photoprism#2612] [Ideas]
I would like to find every photo with one single face, and when this face is not recognized.

I am able to:
- find photos with unrecognized faces ("faces:new")
- find photos with more than 3 faces ("faces:3")

But I can't:
- find photos with more than 3 faces and at least one new face ("faces:new faces:3 doesn't work)
- find photos with exactly one face and one new face (my original issue)

Is it possible or should I open an issue ?

**Comment by graciousgrey** [comment]
You are right, this is not possible at the moment.

faces:new shows all photos with at least one unnamed face

faces:2 shows all photos with at least two faces
